templates/house_view/mine_chart.html

```diff
@@ -0,0 +1,29 @@
+{% load static %}
+
+<script type="text/javascript" src="{% static 'js/random_colors.js' %}" ></script>
+<script>
+    $(document).ready(function () {
+        window.randomColors = [{%for amount in amounts%} randomColor(),{%endfor%}];
+        window.pie_chart = "right_chart"
+        window.bar_chart = "left_chart"
+    });
+</script>
+
+{{ categories|json_script:"categories" }}
+{{ amounts|json_script:"amounts" }}
+
+<script src="{%static 'js/pie_chart.js'%}" type="text/javascript"></script>   
+<script src="{%static 'js/bar_chart.js'%}" type="text/javascript"></script>   
+
+<body>
+  <div>
+    <div class="chart_card">
+      <div class="chart_box">
+        <canvas id="right_chart"></canvas>
+      </div>
+      <div class="chart_box">
+        <canvas id="left_chart"></canvas>
+      </div>
+    </div>
+  </div>
+</body>
```

Comment:
Nicely done! :)

Reply:
please take a look at my new PR https://github.com/redhat-beyond/Xpense/pull/132
this refactored the way of use of the charts and made them more customizable, this way you can change the title or any things you like in the charts by adding more params to the js functions.
please consider cherry pick my branch and add those changes to this PR and modify your use of the charts in the mine page.

Reply:
Nice idea @shlior7 I'll use it here 👍